presto-main/src/main/java/com/facebook/presto/operator/SortedPositionLinks.java

```diff
@@ -165,7 +168,7 @@ private SortedPositionLinks(PositionLinks positionLinks, int[][] sortedPositionL
         this.positionLinks = requireNonNull(positionLinks, "positionLinks is null");
         this.sortedPositionLinks = requireNonNull(sortedPositionLinks, "sortedPositionLinks is null");
         this.lessThanFunction = requireNonNull(lessThanFunction, "lessThanFunction is null");
-        this.sizeInBytes = positionLinks.getSizeInBytes() + sizeOf(sortedPositionLinks);
+        this.sizeInBytes = INSTANCE_SIZE + positionLinks.getSizeInBytes() + (SizeOf.sizeOf(sortedPositionLinks[0]) * sortedPositionLinks.length);
```

Comment:
Actually we can significantly overestimate with the previous version (e.g., int[1000][] and only the first few arrays are non-null and the rest are null). For now I traversed the array.